### ISSUE TITLE
Missing video: "Applied Accessibility: Add an Accessible Date Picker"

### ISSUE THREAD

**Issue by mrmivo** [freeCodeCamp/freeCodeCamp#34926]
The video for the [Applied Accessibility: Add an Accessible Date Picker](https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/add-an-accessible-date-picker/) lecture does not exist.

**Comment by tspeed90** [comment]
Two videos from that section are missing: "Add an Accessible Date Picker" and "Make Elements Only Visible to a Screen Reader by Using Custom CSS". It seems both videos are down on the Scrimba platform. In the [Scrimba playlist](https://scrimba.com/playlist/pJ5rC6), both videos seem to be missing. If this video has been uploaded elsewhere, we can replace the URL for now. Otherwise, those two videos may need to be re-uploaded.

Side note: The playlists are also incorrectly named. It seems that the Applied Accessibility and Applied Visual Design titles have accidentally been switched.

**Comment by ahmadabdolsaheb** [comment]
@beaucarnes
